### ISSUE TITLE
Avoiding the Pitfalls of Deep Learning: Solving Model Overfitting with Regularization and Dropout

### ISSUE THREAD

**Issue by sumankanukollu** [HydPy/HydPy-meetups#45]
**Title of the talk/workshop**
# Avoiding the Pitfalls of Deep Learning: Solving Model Overfitting with Regularization and Dropout

**Abstract of the talk/workshop**
Understanding how to create a deep learning neural network is an essential component of any data scientist's knowledge base. This talk covers some of the challenges that arise when training neural networks. It focuses on the problem of overfitting and its potential remedy: regularization. Learners should have a basic understanding of linear algebra and calculus at a minimum level.

* Discover what overfitting means and how to recognize it in deep learning models
* Understand how to sample your data to reduce the likelihood of overfitting
* Learn about regularization and its use as a remedy for overfitting

**Category of the talk/workshop**

* Data Science, Machine Learning, AI and Neural Networks

**Duration (including Q&A)**

60 minutes

**Level of Audience**

* Internediate level
* Assuming audience have a basic knowledge of neural network Concepts
* In addition to this I'm assuming that you do have a basic understanding of linear algebra and calculus but we won't be using anything more complicated than a derivative and a vector product :) 

**Speaker Bio**

A software professional and community speaker with 12+ years of rich experience in Automation & Tool development of software applications using Python, Django & Flask. Currently, working as a Sr. Software Engineer at F5, in Cyber Security Domain and working on F5 Distributed Cloud Bot Defense solution.

An AI/ML enthusiast with an interest in Deep Learning and Computer Vision and a promoter of DL framework PyTorch.

Lots of desire to learn new exciting things!

**Prerequisites(if any)**

* I recommend to watch the video : [Deep Learning for Beginners (From Basics to Neural Networks)](https://www.youtube.com/watch?v=IuNEfmuoEuM&list=PLqYDykjFMcnGPLPC7cBp9zqgXaRqoUJrC&ab_channel=Codementor) 
[![Final video of fixing issues in your code in VS Code](https://img.youtube.com/vi/IuNEfmuoEuM/maxresdefault.jpg)](https://www.youtube.com/watch?v=IuNEfmuoEuM&list=PLqYDykjFMcnGPLPC7cBp9zqgXaRqoUJrC&ab_channel=Codementor)

**Comment by sumankanukollu** [comment]
Thank you for your response Bhushan.

I felt happy after seeing your email.
Sorry to say, i won’t be in Hyderabad during that weekend.

But i wanted to deliver this content in any of our next event, please
consider this as a request and let me know your response.

**Comment by bpkapkar** [comment]
@sumankanukollu  I am writing to inform you that we have rescheduled our meetup from 1st April to 6th May. I wanted to confirm your availability for the new meetup date, which will be held on 6th May at Hyderabad. Please let me know if this new schedule works for you.

**Comment by sumankanukollu** [comment]
Yes, it works for me, Bhushan. Thanks for the response.

Regards,
Suman Kanukollu
Mobile No.: +91 9703049427
https://www.linkedin.com/in/suman-kanukollu

**Comment by bpkapkar** [comment]
I am writing to inform you that we have rescheduled our meetup from 6th May from 13th May. As you already confirm availability for the new meetup date, which will be held on 13th May at Hyderabad. We are writing to request for brief summary about a you @sumankanukollu  and have a photo of you to share with audience attending the talk. Would it be possible for you to provide requested information and allowing to use the same. Thank you for your time and consideration. We look forward to hearing back from you soon.

**Comment by sumankanukollu** [comment]
Thanks @Bhushan for your message.

Here is my brief summary, please find the attached picture, let me know if
anything is required.
Request you to please share the venue and further details as well.

""" Vamsi Suman, is a Sr. Software Engineer at F5 working in the Cyber
Security domain, specializing in Cloud Automation and tool development
using Python.
With over 12 years of experience, he is passionate about AI/ML,
particularly in the areas of Deep Learning and Computer Vision.
He is also a blogger and community speaker, sharing his insights on topics
such as Deep Learning, AWS, PyTorch, and REST API.
Vamsi is committed to continuous learning and is always eager to expand his
knowledge and skills.
"""

Regards,
Suman Kanukollu
Mobile No.: +91 9703049427
https://www.linkedin.com/in/suman-kanukollu

**Comment by bpkapkar** [comment]
@sumankanukollu  Thanks for sharing your brief summary. I am unable to see the photo in attachment, Will need a photo of you to share with audience attending the talk. Would it be possible for you to provide that.

**Comment by sumankanukollu** [comment]
Not sure why you are not able to see the attached photo.
Here I reattached and same you can find it from here as well, click here
<https://drive.google.com/file/d/1H-CnJRFIVTdxhot-enOWdmNe5UQcQa6b/view?usp=share_link>

Regards,
Suman Kanukollu
Mobile No.: +91 9703049427
https://www.linkedin.com/in/suman-kanukollu
